gocourse05/camera/day_test.go

```diff
@@ -0,0 +1,58 @@
+package camera
+
+import (
+	"testing"
+)
+
+func TestDayCameraRetrieveData(t *testing.T) {
+	t.Run("Found", func(t *testing.T) {
+		c := NewDayCamera(1, "Canon", []Data{NewDayCameraData(1, "Коза", "Побігла вліво")})
+
+		data, err := c.RetrieveData()
+		if err != nil {
+			t.Errorf("RetrieveData() should not have error: %s", err)
+		}
+
+		if len(data) == 0 {
```

Comment:
Краще явно перевіряти, що очікується рівно 1 елемент в слайсі:
```suggestion
		if len(data) != 1 {
```

Тому що якщо буде 2, то тест спокійно пройде.